### DISCUSSION TITLE
Informatics for RNA Sequencing

### DISCUSSION THREAD

**Discussion by josesanchez1815** [nih-cfde/training-and-engagement#450] [RNA - Seq] [answered]
So this question comes in response to slide 25 of 91 from these google slides https://osf.io/kj5av/

So is it like a flow cell sorter or does it make the library in the wells?

**Comment by ACharbonneau** [comment, marked as answer]
You make your library before you get anywhere near a sequencer. Flow cells are coated with stuff that allows the Illumina primers to stick. When you wash your sequencing sample library into a flow cell, there's no sorting or anything, just random annealing. Not every strand in your library will get sequenced. Usually you only load about half of the library anyway, only some of that actually sticks, the rest washes away before sequencing starts, and of those that stick, only some will have high enough quality to read.
